src/index.css

```diff
@@ -177,125 +177,109 @@
     -webkit-text-fill-color: transparent;
   }
 
-  .h1_32_40_heavy {
+  .text_h1_heavy {
```

Comment:
I know underscore may look more readable. But please let's use hyphen `-` anyway, since all other classes are named in such way.

Reply:
Updated